lib/api/controllers/bulkController.js

```diff
@@ -128,31 +118,26 @@ class BulkController extends BaseController {
       { refresh, injectKuzzleMeta: false }
     )
       .then(({ items, errors }) => {
-        if (errors.length > 0) {
-          request.setError(
-            errorsManager.get('document_creations_failed', errors));
-        }
-
         if (notify) {
           this.kuzzle.notifier.notifyDocumentMChanges(
             request,
             items,
             true);
         }
 
-        const hits = [];
+        const documents = [];
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') "documents" hides or potentially hides a variable declared in an outer scope at line 110. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableShadowing)